pandas/core/frame.py

```diff
@@ -10605,11 +10605,12 @@ def corrwith(
                             0, 1
                         ]
                 else:
+                    from scipy.stats import rankdata
```

Comment:
Unfortunately, this cannot be used since scipy is still considered an optional dependency and the function was usable before without the dependency installed.

Reply:
Ok, that's fine. I will use `algos.rank_1d`.